nodes/src/nodes/eval_cobalt/cobalt_evaluator.py

```
@@ -0,0 +1,265 @@
+      - LLM-as-judge (GPT-4 / Claude scoring with criteria)
+      - custom function (arbitrary Python callable returning a score)
+    """
+
+    _eval_type: str
+    _threshold: float
+    _model: str
+    _criteria: str
+    _apikey: str
+    _custom_fn: Optional[Callable] = None
+
+    def __init__(self, config: Dict[str, Any], bag: Dict[str, Any]) -> None:
```

Ruff (0.15.7)

[warning] Unused method argument: `bag`

(ARG002)